docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>I think those should either be accompanied of a quick example (i.e. prior sampling plus inspecting the inferencedata result) or adding links to some relevant pages (i.e. the prior/posterior predictive one)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Agreed. Im not going to do it in this PR. I'll open an issue and open it up to others or get back to it later so we dont lose track</p>